doc/dev/CredscanProcess.md

```diff
@@ -0,0 +1,66 @@
+If your PR fails the Build Compliance check, follow these steps: 
+
+1. Click “View more details on Azure Pipelines”. In the “Compliance” check on the Pipeline page, view the “Scanning for credentials” step. This page will show where the leaked secrets are in the current PR.
+
+2. Each warning will begin with the path to the file containing a potential credential, as well as the row and column where
+the credential string begins. For example, for a potential credential that starts in row 3 and column 20 of a
+particular file:
+```
+##[error]sdk/{service}/{package}/{file}.json:sdk/{service}/{package}/{file}.json(3,20)
+```
+
+The warning will then list an error code and description of why the potential credential was flagged.
+
+## True Positives
+Depending on if you are working on a Track 1 service or a Track 2 service there are different steps to adding/modifying the sanitizer:
+
+For a Track 1 Service:
+1.	Access your service's TestBase.cs file.
+2.	In the static constructor, JsonPathSanitizers can be added. This allows the sanitization of particular fields identified such as “accessSAS”. Add a field to sanitize like so: “RecorderUtilities.JsonPathSanitizers.Add("$..{FIELDNAMEHERE");”
+3.	An example can be found in here in the Track 1 Compute [TestBase.cs file](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/VMTestBase.cs#L32).
+
+For a Track 2 Service:
+1.	Access your service's [RPName]ClientBase.cs file. It should extend the ManagementRecordedTestBase. An example for the  CosmosDBManagementClientBase class is shown [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/CosmosDBManagementClientBase.cs).
+2. Check the proctected constructor in your [RPName]ClientBase.cs file. If a [RPName]RecordedTestSanitizer object is initialized here, open that file. Otherwise, create a [RPName]RecordedTestSanitizer.cs file that extends the RecordedTestSanitizer class. An example of the CosmosDBRecordedTestSanitizer.cs can be found [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/CosmosDBManagementRecordedTestSanitizer.cs). 
+3. Add the fields that need to be sanitized into the [RPName]RecordedTestSanitizer.cs file into the constructor as shown [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/CosmosDBManagementRecordedTestSanitizer.cs#L14).
+3.	If the protected constructor in your [RPName]ClientBase class does not create a [RPName]RecordedTestSanitizer object, add the initialization [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/CosmosDBManagementClientBase.cs#L48).
+
+Once you have added the Sanitizers, re-record all your test files that had leaked secrets. They should be replaced with the keyword “Sanitized”.
+## False Positives
```

Comment:
I think the sanitizer approach also applies some false positive cases.

Reply:
Moved to internal